controls/radrichtextbox/features/positioning.md

```diff
@@ -9,72 +9,145 @@ published: True
-      
 
 >When using the __CaretPosition__ property you are directly managing the caret position in the __RadDocument__. By using the __DocumentPosition__ class you can create instances of several positions inside the document without changing the current caret position.
-        
+
+* [Access document element from caret position](#access-document-element-from-caret-position)
+* [Convert between DocumentPosition and screen position](#convert-between-documentposition-and-screen-position)
 
 ## CaretPosition property
 
 Here is an example of how to use the __CaretPosition__ property to get the current word.
-        
-
-#### __C#__
 
+#### __[C#] Example 1: Getting the word on the caret position__  
 {{region radrichtextbox-features-positioning_0}}
 	string currentSpanText = this.radRichTextBox.Document.CaretPosition.GetCurrentSpanBox().Text;
 {{endregion}}
 
-
-
-#### __VB.NET__
-
+#### __[VB.NET] Example 1: Getting the word on the caret position__  
 {{region radrichtextbox-features-positioning_1}}
 	Dim currentSpanText As String = Me.radRichTextBox.Document.CaretPosition.GetCurrentSpanBox().Text
 {{endregion}}
 
-
-
 ## DocumentPosition class
 
-An alternative of using the __CaretPosition__ property is to create an instance of the __DocumentPosition__ class. Here is the same example from the previous chapter done with an instance of the __DocumentPosition__ class.
-        
+An alternative of using the __CaretPosition__ property is to create an instance of the __DocumentPosition__ class. Here is the same example from the previous chapter done with an instance of the __DocumentPosition__ class.        
 
 >When instantiated in such a way, the position will be at the start of the document, so the result will be the first word in the text.The position of the instance won't be affected by the UI. To change it you have to use {% if site.site_name == 'Silverlight' %}[the API of the DocumentPosition class](http://www.telerik.com/help/silverlight/t_telerik_windows_documents_documentposition.html){% endif %}{% if site.site_name == 'WPF' %}[the API of the DocumentPosition class](http://docs.telerik.com/devtools/wpf/api/html/t_telerik_windows_documents_documentposition.htm){% endif %}.
             
 
-#### __C#__
-
+#### __[C#] Example 2: Getting the word on the caret using the DocumentPosition class__  
 {{region radrichtextbox-features-positioning_2}}
 	DocumentPosition position = new DocumentPosition( this.radRichTextBox.Document );
 	string currentSpanText = position.GetCurrentSpanBox().Text;
 {{endregion}}
 
-
-
-#### __VB.NET__
-
+#### __[VB.NET] Example 2: Getting the word on the caret position using the DocumentPosition class__  
 {{region radrichtextbox-features-positioning_3}}
 	Dim position As New DocumentPosition(Me.radRichTextBox.Document)
 	Dim currentSpanText As String = position.GetCurrentSpanBox().Text
 {{endregion}}
 
+## Access document element from caret position

```

Comment:
Would be useful to mention that using the DocumentPosition, the customer can also set start and end of a selection range. Link http://docs.telerik.com/devtools/wpf/controls/radrichtextbox/features/selection#programmatic-selection